### PR TITLE
DOC-11487: Updated transactions/XDCR text

### PR DESCRIPTION
**Pull request by tonyjhillman**

### CODE REVIEW

File: modules/learn/pages/data/transactions.adoc [hunk 1]
```diff
@@ -172,11 +172,21 @@ Note that you can choose to update the scan consistency level to `not_bounded` i
 
 === Transactions and Replication (XDCR)
 
-xref:learn:clusters-and-availability/xdcr-overview.adoc[Cross Data Center Replication] (XDCR) supports eventual consistency of transactional changes. No uncommitted changes will be ever sent to the target clusters.  Once committed, the transactional changes will arrive one by one at the target. If the connection is lost midway while receiving a transaction it is possible for the target to receive a partial transaction.
+xref:learn:clusters-and-availability/xdcr-overview.adoc[Cross Data Center Replication (XDCR)] supports eventual consistency of transactional changes.
+No uncommitted changes will be ever sent to target clusters.
+Once committed, the transactional changes arrive one by one at the target.
+If the connection is lost midway, it is possible for the target to receive a partial transaction.
```
Comment (nelio2k) [inline modules/learn/pages/data/transactions.adoc:178]: I don't think this is possible. Considering transactions in a single document at this point: When a transaction is started, and before it is committed, the uncommitted transaction will live in a temporary xattr metadata wihtin the document. XDCR will not replicate this over. Thus, it's not possible for the target to receive anything *but* a document after the transaction is completed.

Comment (hyunjuV) [reply modules/learn/pages/data/transactions.adoc:178]: If the transaction involves multiple documents, then, it is possible for the target to receive a partial transaction (in the scenario that the connection is lost midway), isn't it?   I think that's all it is saying.

**Review by hyunjuV — APPROVED** [review]

File: modules/learn/pages/data/transactions.adoc (hunk 1 above)
Comment (hyunjuV) [reply modules/learn/pages/data/transactions.adoc:178]: ^^ @nelio2k -- can you confirm?  If a transaction involves multiple documents, once the transaction is committed, xdcr does not guarantee that all of the documents in a transaction will be replicated at the same time, and therefore, if the connection is lost, the customer could see a partial transaction on the target -- where document A was replicated before the connection was lost, but document B was not.

Comment (nelio2k) [reply modules/learn/pages/data/transactions.adoc:178]: Correct

**Pull request merged**